test/rotation_matrix_test.cpp

```diff
@@ -0,0 +1,250 @@
+#include "surface_perception/shape_extraction.h"
+#include "Eigen/Eigen"
+
+#include "pcl/ModelCoefficients.h"
+
+#include <gtest/gtest.h>
+namespace {
+// This test the equality of two matrix through approximation.
+bool isSameMatrix(const Eigen::Matrix3f& actual_matrix,
```

Comment:
Use Eigen's `isApprox` instead of `isSameMatrix`. See https://github.com/jstnhuang/transform_graph/blob/indigo-devel/test/orientation_test.cpp for examples.

Reply:
Completed